Mathlib/CategoryTheory/Limits/FunctorCategory.lean

```diff
@@ -321,7 +321,7 @@ theorem colimit_obj_ext {H : J ⥤ K ⥤ C} [HasColimitsOfShape J C] {k : K} {W
 
 instance evaluationPreservesLimits [HasLimits C] (k : K) :
     PreservesLimits ((evaluation K C).obj k) where
-  preservesLimitsOfShape {J} 𝒥 := by skip; infer_instance
+  preservesLimitsOfShape {_} _𝒥 := inferInstance
```

Comment:
Doesn't `inferInstance` add extra junk that `by infer_instance` doesn't? Can you check the term is syntactically the same as it was before?

Reply:
In its current form (with `inferInstance`):
```lean
#print evaluationPreservesLimits

def CategoryTheory.Limits.evaluationPreservesLimits.{v₂, u₂, v, u} : {C : Type u} →
  [inst : Category.{v, u} C] →
    {K : Type u₂} →
      [inst_1 : Category.{v₂, u₂} K] → [inst_2 : HasLimits C] → (k : K) → PreservesLimits ((evaluation K C).obj k) :=
fun {C} [Category.{v, u} C] {K} [Category.{v₂, u₂} K] [HasLimits C] k ↦
  { preservesLimitsOfShape := fun {x} _𝒥 ↦ inferInstance }
```
As it was before the change:
```lean
#print evaluationPreservesLimits

def CategoryTheory.Limits.evaluationPreservesLimits.{v₂, u₂, v, u} : {C : Type u} →
  [inst : Category.{v, u} C] →
    {K : Type u₂} →
      [inst_1 : Category.{v₂, u₂} K] → [inst_2 : HasLimits C] → (k : K) → PreservesLimits ((evaluation K C).obj k) :=
fun {C} [Category.{v, u} C] {K} [Category.{v₂, u₂} K] [HasLimits C] k ↦
  { preservesLimitsOfShape := fun {J} 𝒥 ↦ inferInstance }
```
Seems that the only difference are the variable names.